gridwords.py

```diff
@@ -19,6 +20,12 @@
 from move import moveUp, moveDown, moveLeft, moveRight, select, highlight
 from datasearch import getPossWords, allPossWords
 
+LOGGER_FORMAT = "%(filename)s:%(lineno)s %(funcName)s: %(message)s"
+LOGGER_LEVEL = logging.INFO
+#LOGGER_LEVEL = logging.DEBUG
+logging.basicConfig( format=LOGGER_FORMAT, level=LOGGER_LEVEL)
+logger = logging.getLogger(__name__)
```

Comment:
Control the level of logging in this file by uncommenting the correct `LOGGER_LEVEL`.